card/src/main/java/com/adyen/checkout/card/internal/ui/model/InstallmentOption.kt

```diff
@@ -8,7 +8,10 @@
 
 package com.adyen.checkout.card.internal.ui.model
 
-internal enum class InstallmentOption(val type: String?) {
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Let's not expose this class just to use it in the example app, we can hardcode the value we want to use there